GOVERNANCE.md

```diff
@@ -43,30 +57,32 @@ Have you done something (big or small) to contribute to the health, success, or
 
 #### Responsibilities
 
-This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
+This role does not require any extra responsibilities nor time commitment. We hope you stick around and keep participating in our community!
 
-If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
+If you're interested in reaching the next level and becoming a **Maintainer**, you can explore some of those responsibilities in the [next section](#level-2-l2---maintainer).
 
 #### Nomination Process
 
-If you contributed to Astro outside of Discord, you may self-nominate by sending `!contribute` in any Discord channel, accompanied by a link or description of your contribution. You may also gain this role if you are active and helpful on Discord.
+You may self-nominate by sending the message `!contribute` in any Discord channel. If you do this, please share a second message with a link or description of your contribution so that people can recognize you for the contribution,
```

Comment:
```suggestion
You may self-nominate by sending the message `!contribute` in any Discord channel. If you do this, please share a second message with a link or description of your contribution so that people can recognize you for the contribution.
```